WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -562,17 +562,31 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleElementScreenshot:(FBRouteRequest *)request
 {
-  FBElementCache *elementCache = request.session.elementCache;
-  XCUIElement *element = [elementCache elementForUUID:(NSString *)request.parameters[@"uuid"]
-                                       checkStaleness:YES];
-  NSData *screenshotData = [element.screenshot PNGRepresentation];
-  if (nil == screenshotData) {
-    NSString *errMsg = [NSString stringWithFormat:@"Cannot take a screenshot of %@", element.description];
-    return FBResponseWithStatus([FBCommandStatus unableToCaptureScreenErrorWithMessage:errMsg
-                                                                             traceback:nil]);
+  @autoreleasepool {
```

Comment:
We cannot accept this change because the webdriver spec [requires](https://www.w3.org/TR/webdriver1/#screen-capture) to provide screenshots in PNG format and not any other.

Also, these screenshots may be used for OCR, image comparison, etc, where lossless compression is important because it significantly improves the quality and reliability of above processes